lib/std/stropt.nim

```diff
@@ -0,0 +1,57 @@
+import std/strutils
```

Comment:
the other way around makes more sense as argued in https://github.com/nim-lang/Nim/pull/16280#issuecomment-740084520

strutils should import `stropt` and (if needed) publicly re-export some symbols to keep it DRY

Reply:
Do you think `stripInplace` should be renamed?

Reply:
yes, given that new convention in nim since `dup` is to prefer inplace procs.

So renaming to `strip` is fine, especially now that it's not in same module as strutils.strip